src/rust/ide/src/ide/integration.rs

```diff
@@ -289,19 +289,20 @@ impl Integration {
         }
 
         frp::extend! { network
-            eval_ editor_outs.node_added([]{analytics::remote_log(analytics::AnonymousData("node_added"))});
-            eval_ editor_outs.node_removed([]{analytics::remote_log(analytics::AnonymousData("node_removed"))});
-            eval_ editor_outs.nodes_collapsed([]{analytics::remote_log(analytics::AnonymousData("nodes_collapsed"))});
-            eval_ editor_outs.node_entered([]{analytics::remote_log(analytics::AnonymousData("node_entered"))});
-            eval_ editor_outs.node_exited([]{analytics::remote_log(analytics::AnonymousData("node_exited"))});
-            eval_ editor_outs.node_exited([]{analytics::remote_log(analytics::AnonymousData("node_exited"))});
-            eval_ editor_outs.on_edge_endpoints_set([]{analytics::remote_log(analytics::AnonymousData("edge_endpoints_set"))});
-            eval_ editor_outs.visualization_enabled([]{analytics::remote_log(analytics::AnonymousData("visualization_enabled"))});
-            eval_ editor_outs.visualization_disabled([]{analytics::remote_log(analytics::AnonymousData("visualization_disabled"))});
+            eval_ editor_outs.node_editing_started([]{analytics::remote_log(analytics::AnonymousData("editor_node_editing_started"))});
+            eval_ editor_outs.node_editing_finished([]{analytics::remote_log(analytics::AnonymousData("editor_node_editing_finished"))});
+            eval_ editor_outs.node_added([]{analytics::remote_log(analytics::AnonymousData("editor_node_added"))});
+            eval_ editor_outs.node_removed([]{analytics::remote_log(analytics::AnonymousData("editor_node_removed"))});
+            eval_ editor_outs.nodes_collapsed([]{analytics::remote_log(analytics::AnonymousData("editor_nodes_collapsed"))});
+            eval_ editor_outs.node_entered([]{analytics::remote_log(analytics::AnonymousData("editor_node_entered"))});
```

Comment:
This is a node enter request right? This des not mean the node is really entered right? If so, I'd rename "editor_node_entered" to "editor_node_enter_request"

Reply:
I'd rather keep it with the name of the actual function here.

Reply:
Why? I think the functionshould also be renamed, but lets do it later. The logged things doesnt make sense now really, as they use past form "entered", while its not true at all. We will definitely rename the graph editor events, but then we would need to keep wrong logging event names. I'd prefer to use correct logging naes and later update the graph instead.